cfme/tests/infrastructure/test_advanced_search_host.py

```diff
@@ -253,6 +252,5 @@ def test_cannot_delete_more_than_once(host_collection, hosts_advanced_search,
     if hosts_advanced_search.entities.search.delete_filter():
         # This should not happen
         msg = "Delete twice accepted!"
-        if is_cfme_exception():
-            msg += " CFME Exception text: `{}`".format(cfme_exception_text())
+        hosts_advanced_search.flash.assert_no_error()
```

Comment:
**Required:** Text of flash message should be checked here. hosts_advanced_search.flash.assert_message(' Text of flash message here ')

Reply:
@ohiliazo Sorry mislead you as there is no any flash message on UI. So for verification, we can check if can delete filter second time by use: 
assert not hosts_advanced_search.entities.search.delete_filter(), 'Your custom message here'